splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceIndexWatcher.java

```diff
@@ -31,7 +31,7 @@
     protected String tableName;
     protected String tableSchemaName;
     protected String createString;
-    protected String indexName;
+    public final String indexName;
```

Comment:
Nitpick, a getter for this would be better

Reply:
fixed it.